packages/web-sdk/src/transports/otlp/payload/transform/types.ts

```diff
@@ -0,0 +1,57 @@
+import type { APIEvent, TransportItem } from '@grafana/faro-core';
+
+import type { Attribute } from '../attribute';
+
+export interface ResourcePayload {
```

Comment:
I was looking for typescript types in the reference sdk. There's https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/api-logs/src/types/LogRecord.ts, but I'm not sure these work as export request types, and there's no log exporter yet in there